projects/github-actions/repo-gardening/changelog/update-gardening-action-extract

```diff
@@ -0,0 +1,4 @@
+Significance: major
+Type: added
+
+Repo Gardening: initial public release for external consumption
```

Comment:
I thought 2.0.0 was different enough (packaged on its own) that it deserved its own major release, so 2.0.0 made sense to me that way. But since I'm not actually adding any new functionality, you're right that a small version update may make more sense.

Reply:
Oh, if you want to start at 2.0.0 by counting the unreleased version as 1.0.0, that makes sense.